app/services/file_attachment_upload_service.rb

```diff
@@ -40,4 +45,12 @@ def filename
     existing_filenames = revision.file_attachment_revisions.map(&:filename)
     UniqueFilenameService.new(existing_filenames).call(file.original_filename)
   end
+
+  def page_count
+    return unless file.content_type == PDF_CONTENT_TYPE
```

Comment:
dare say at this point we might not need a constant since we're only using it here

Reply:
Might it make sense to make the constant an array to hold PDF and any future content types that might have pages?